hr_holidays_natural_period/tests/test_hr_leave.py

```diff
@@ -82,14 +83,25 @@ def test_hr_leave_natural_day_01(self):
         self.assertEqual(res_leave_type["max_leaves"], 5)
         self.assertEqual(res_leave_type["leaves_taken"], 0)
         self.assertEqual(res_leave_type["virtual_leaves_taken"], 0)
-        self.assertEqual(res_leave_type["request_unit"], "natural_day")
+        self.assertEqual(res_leave_type["request_unit"], self.leave_type.request_unit)
         leave = self._create_hr_leave(self.leave_type, "2023-01-02", "2023-01-05")
-        self.assertEqual(leave.number_of_days, 4.0)
-        self.assertEqual(leave.number_of_days_display, 4.0)
+        if self.leave_type.request_unit == "natural_day":
```

Comment:
The test should be self-determinant, so why this if?

Reply:
Okay, changed and avoided the if statements in the tests; each test checks the expected values.